### ISSUE TITLE
the server is down!

### ISSUE THREAD

**Issue by elijahchancey** [sqren/freedom#7]
http://freedom.konscript.com/

**Comment by jacekzwroclawia** [comment]
+ 1

**Comment by svict4** [comment]
His whole website is down.
When Facebook updated their API a while ago, it kinda broke Freedom anyway :cry:

**Comment by sqren** [comment]
Yeah, sorry about that. Facebook has changed their api drastically, and it is now harder to get the permissions required.
